Source/WebInspectorUI/UserInterface/Views/LocalResourceOverridePopover.js

```diff
@@ -77,6 +77,12 @@ WI.LocalResourceOverridePopover = class LocalResourceOverridePopover extends WI.
             const schemes = ["http:", "https:", "file:"];
             if (!schemes.some((scheme) => data.url.toLowerCase().startsWith(scheme)))
                 return null;
+        } else {
```

Comment:
NIT: it'd probably flip the order of this `if` so that the `isRegex` check is closer to the `RegExp` usage
```js
if (data.isRegex) {
    try {
        new RegExp(data.url);
    } catch {
        return null;
    }
} else {
    const schemes = ["http:", "https:", "file:"];
    if (!schemes.some((scheme) => data.url.toLowerCase().startsWith(scheme)))
        return null;
}
```
apologies for not suggesting this before.  didn't realize we already had a check for `data.isRegex` nearby

Reply:
Funny enough, I consider changing it like this the first time around, but I hesitated 😄 
Updated now.